helm/linkerd-control-plane/values.schema.json

```diff
@@ -88,7 +88,7 @@
                             "type": "string"
                         },
                         "maxUnavailable": {
-                            "type": ["integer","string"]
+                            "type": "integer"
```

Comment:
Both `maxSurge` and `maxUnavailable` can be either an integer or a string since it's also possible to define percentage.

Reply:
you are right sir. thanks!